### PR TITLE
Add image info

### PR DESCRIPTION
**Pull request by N0U**
Uploadcare send image info structure which lib doesn't read. So I fixed it.

### CODE REVIEW

File: src/main/java/com/uploadcare/api/Image.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import com.uploadcare.data.ImageInfo;
+
+import java.util.Date;
+
+/**
+ * Created by Egor Petushkov
+ * on 09.06.17.
+ */
+public class Image extends File {
+    public static final int UNKNOW_DPI = -1;
```
Comment (dmitry-mukhin) [inline src/main/java/com/uploadcare/api/Image.java]: unknown?

File: src/main/java/com/uploadcare/api/Image.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    public enum Orientation {
```
Comment (dmitry-mukhin) [inline src/main/java/com/uploadcare/api/Image.java]: I don't think that this adds any value vs using numbers.
That's why you need those lengthy comments :)
Can you please elaborate to defend this?

File: src/main/java/com/uploadcare/api/Image.java (hunk 1 above)
Comment (N0U) [inline src/main/java/com/uploadcare/api/Image.java]: I wanted to use it when dpi isn't supplied, but now I think I should use null instead.

Comment (dmitry-mukhin) [inline src/main/java/com/uploadcare/api/Image.java]: I mean there's a typo: `UNKNOW_DPI` -> `UNKNOWN_DPI`

Comment (dmitry-mukhin) [inline src/main/java/com/uploadcare/api/Image.java]: or better yet, `DPI_UNKNOWN`

File: src/main/java/com/uploadcare/api/Image.java (hunk 2 above)
Comment (N0U) [inline src/main/java/com/uploadcare/api/Image.java]: System is pretty simple when you get it.
There are 8 orientations which you can split on 2 groups. The first group is when you take an image and rotate it clockwise. The second group is when you horizontally flip an image and rotate it clockwise. Illustration: http://www.impulseadventure.com/photo/images/orient_flag.gif
So once you understand how it works, these names makes sense.

Comment (dmitry-mukhin) [inline src/main/java/com/uploadcare/api/Image.java]: But do you need those as lib user? I don't feel that way at the moment.
If you're going to work with orientation value, you already should know what that means.

Comment (N0U) [inline src/main/java/com/uploadcare/api/Image.java]: I don't think so, maybe later. I think it is useful for small programs where just request image form uploadcare, check size, format, orientation and send it to the front end.

**Comment by rsedykh** [comment]
Should be closed with #28.

**Pull request closed without merging**